tf2_ros_py/package.xml

```diff
@@ -0,0 +1,46 @@
+<package format="2">
+  <name>tf2_ros_py</name>
+  <version>0.12.4</version>
+  <description>
+    This package contains the ROS Python bindings for the tf2 library
+  </description>
+  <author>Eitan Marder-Eppstein</author>
+  <author>Wim Meeussen</author>
+  <maintainer email="tfoote@osrfoundation.org">Tully Foote</maintainer>
+  <license>BSD</license>
+
+  <url type="website">http://www.ros.org/wiki/tf2_ros</url>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+  <buildtool_depend>ament_cmake_python</buildtool_depend>
+  <buildtool_depend>python_cmake_module</buildtool_depend>
+
+  <build_depend>geometry_msgs</build_depend>
+  <build_depend>std_msgs</build_depend>
+
+  <exec_depend>geometry_msgs</exec_depend>
+  <exec_depend>std_msgs</exec_depend>
+
+  <depend>message_filters</depend>
+  <depend>rclcpp</depend>
+  <depend>rclcpp_action</depend>
+  <depend>rclcpp_components</depend>
+  <depend>rclpy</depend>
+  <depend>tf2</depend>
+  <depend>tf2_msgs</depend>
+  <depend>tf2_py</depend>
+  <depend>tf2_ros</depend>
+
+  <!-- TODO(tfoote) dependencies from unported sections restore when ported
+  <depend>xmlrpcpp</depend> -->
+  <!-- <build_depend>rosgraph</build_depend> -->
+
+  <!-- <test_depend>rostest</test_depend> -->
```

Comment:
I think we can just remove these at this point.  `xmlrpcpp` will never be a thing in ROS 2, the testing infrastructure is different (rostest goes away), and we'll reintroduce rosgraph if we need it.